apps/dot-voting/app/components/Card/Vote.js

```diff
@@ -62,25 +66,19 @@ class Vote extends React.Component {
     }
 
     return (
-      <StyledCard>
-        <div onClick={this.handleVoteClick}>
+      <StyledCard onClick={this.handleVoteClick} css="cursor: pointer">
```

Comment:
Did a linting rule complain about the missing `cursor: pointer`??

Reply:
Also, clever avoidance of the a11y linter 😉 

(Really, though, it's fine for now. We'll revisit this app soon enough!)

Reply:
Linting complained about having onClick slapped onto div without the div having "role" and/or something more official. Normally divs are (as you told me many times! :) just simple structural elements. Div with onClick needs to be equipped with some rules. The way above: code is shorter and lint stopped complaining. Thx :)